pytorch_blade/torch_blade/tensorrt/tensorrt_optimization.py

```diff
@@ -80,4 +88,7 @@ def optimize_trt(script_module, q_info=None, disable_fallback=False):
     dynamic_settings = get_dynamic_settings(c_module, trt_unsupported)
     top_level_block = graph
     supported_node_fusion(graph, top_level_block, trt_unsupported, q_info)
-    trt_engine_conversion(c_module, q_info, disable_fallback, dynamic_settings)
+    with get_calib_file_for_each_group(c_module) as quantization_calib_file:
+        trt_engine_conversion(
+            c_module, q_info, disable_fallback, dynamic_settings, quantization_calib_file
+        )
```

Comment:
remove quantization_calib_file when exiting the context.

Reply:
If leave the context of `with tempfile.TemporaryDirectory(prefix="torch_blade") as tmp_dirname:`, the temporary directory will be automatically removed:)